### PR TITLE
feat(project): add project files from VS

### PR DESCRIPTION
**Pull request by thatglance**
@weterbogov review

### CODE REVIEW

File: GraphicsTemplate/GraphicsTemplate/Circle.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+	public abstract void draw();
+
+	public abstract vector<Point> location();
+
+	/// 
+	/// <param name="destination"></param>
+	public abstract void move(Point destination);
+
+}//end Circle
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/Circle.cs]: @thatglance it seems that you don't have new line at the end of file formatting. Please add this to your settings and fix this in all files

File: GraphicsTemplate/GraphicsTemplate/DoubleDimensional.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		return null;
+	}
+
+	/// 
+	/// <param name="newVal"></param>
+	public void setFillColor(Color newVal){
+
+	}
+
+}//end DoubleDimensional
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/DoubleDimensional.cs]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/Ellipse.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	/// <param name="destination"></param>
+	public abstract void move(Point destination);
+
+	/// 
+	/// <param name="newVal"></param>
+	public void SetPoint(Point newVal){
+
+	}
+
+}//end Ellipse
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/Ellipse.cs]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/Form1.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+namespace GraphicsTemplate
+{
+    public partial class Form1 : Form
+    {
+        public Form1()
+        {
+            InitializeComponent();
+        }
+    }
+}
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/Form1.cs:20]: @thatglance fix end of file

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+
+# LightSwitch generated files
+GeneratedArtifacts/
+ModelManifest.xml
+
+# Paket dependency manager
+.paket/paket.exe
+
+# FAKE - F# Make
+.fake/
```
Comment (mikitabut) [inline .gitignore]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/GraphicsTemplate.csproj [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
+       Other similar extension points exist, see Microsoft.Common.targets.
+  <Target Name="BeforeBuild">
+  </Target>
+  <Target Name="AfterBuild">
+  </Target>
+  -->
+</Project>
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/GraphicsTemplate.csproj]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/IsoscelesTriangle.cs [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+	}
+
+	/// 
+	/// <param name="newVal"></param>
+	public void SetPoint(int newVal){
+
+	}
+
+}//end IsoscelesTriangle
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/IsoscelesTriangle.cs]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/Line.cs [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+	/// <param name="destination"></param>
+	public abstract void move(Point destination);
+
+	/// 
+	/// <param name="newVal"></param>
+	public void SetSecondPoint(Point newVal){
+
+	}
+
+}//end Line
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/Line.cs]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/LineSegment.cs [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+
+	public abstract void draw();
+
+	public abstract vector<Point> location();
+
+	/// 
+	/// <param name="destination"></param>
+	public abstract void move(Point destination);
+
+}//end LineSegment
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/LineSegment.cs]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/OneDimensional.cs [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+
+	~OneDimensional(){
+
+	}
+
+	public override void Dispose(){
+
+	}
+
+}//end OneDimensional
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/OneDimensional.cs]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/Parallelogram.cs [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+	public override vector<Point> location(){
+
+		return null;
+	}
+
+	/// 
+	/// <param name="destination"></param>
+	public abstract override void move(Point destination);
+
+}//end Parallelogram
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/Parallelogram.cs]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/Poligon.cs [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+	/// <param name="destination"></param>
+	public abstract override void move(Point destination);
+
+	/// 
+	/// <param name="newVal"></param>
+	public void setVertexes(vector<Point> newVal){
+
+	}
+
+}//end Poligon
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/Poligon.cs]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/Properties/Resources.resx [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+</root>
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/Properties/Resources.resx]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/Ray.cs [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+
+	public abstract void draw();
+
+	public abstract vector<Point> location();
+
+	/// 
+	/// <param name="destination"></param>
+	public abstract void move(Point destination);
+
+}//end Ray
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/Ray.cs]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/Rectangle.cs [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+	public override vector<Point> location(){
+
+		return null;
+	}
+
+	/// 
+	/// <param name="destination"></param>
+	public abstract override void move(Point destination);
+
+}//end Rectangle
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/Rectangle.cs]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/RegularPolygon.cs [hunk 16]
```diff
@@ -0,0 +1,50 @@
...
+	/// <param name="destination"></param>
+	public abstract override void move(Point destination);
+
+	/// 
+	/// <param name="newVal"></param>
+	public void setCount(int newVal){
+
+	}
+
+}//end RegularPolygon
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/RegularPolygon.cs]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/Rhomb.cs [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+	public override vector<Point> location(){
+
+		return null;
+	}
+
+	/// 
+	/// <param name="destination"></param>
+	public abstract override void move(Point destination);
+
+}//end Rhomb
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/Rhomb.cs]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/Shape.cs [hunk 18]
```diff
@@ -0,0 +1,59 @@
...
+
+	}
+
+	/// 
+	/// <param name="newVal"></param>
+	public void setCenter(Point newVal){
+
+	}
+
+}//end Shape
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/Shape.cs]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/RightTriangle.cs [hunk 19]
```diff
@@ -0,0 +1,50 @@
...
+	/// <param name="destination"></param>
+	public abstract override void move(Point destination);
+
+	/// 
+	/// <param name="newVal"></param>
+	public void setVertex(Point newVal){
+
+	}
+
+}//end RightTriangle
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/RightTriangle.cs]: @thatglance fix end of file

File: GraphicsTemplate/GraphicsTemplate/Circle.cs [hunk 20]
```diff
@@ -0,0 +1,34 @@
+///////////////////////////////////////////////////////////
+//  Circle.cs
+//  Implementation of the Class Circle
+//  Generated by Enterprise Architect
+//  Created on:      05-???-2018 19:15:42
+//  Original author: Nikita
+///////////////////////////////////////////////////////////
```
Comment (mikitabut) [inline GraphicsTemplate/GraphicsTemplate/Circle.cs]: @thatglance remove this strings please from this and another files

**Review by mikitabut — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 5 above)
Comment (thatglance) [reply .gitignore]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/Circle.cs (hunk 20 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/Circle.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/Circle.cs (hunk 1 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/Circle.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/DoubleDimensional.cs (hunk 2 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/DoubleDimensional.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/Ellipse.cs (hunk 3 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/Ellipse.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/Form1.cs (hunk 4 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/Form1.cs:20]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/IsoscelesTriangle.cs (hunk 7 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/IsoscelesTriangle.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/Line.cs (hunk 8 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/Line.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/LineSegment.cs (hunk 9 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/LineSegment.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/OneDimensional.cs (hunk 10 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/OneDimensional.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/Parallelogram.cs (hunk 11 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/Parallelogram.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/Poligon.cs (hunk 12 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/Poligon.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/Ray.cs (hunk 14 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/Ray.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/Rectangle.cs (hunk 15 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/Rectangle.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/RegularPolygon.cs (hunk 16 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/RegularPolygon.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/Rhomb.cs (hunk 17 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/Rhomb.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/RightTriangle.cs (hunk 19 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/RightTriangle.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/Shape.cs (hunk 18 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/Shape.cs]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/Properties/Resources.resx (hunk 13 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/Properties/Resources.resx]: @weterbogov fixed

File: GraphicsTemplate/GraphicsTemplate/GraphicsTemplate.csproj (hunk 6 above)
Comment (thatglance) [reply GraphicsTemplate/GraphicsTemplate/GraphicsTemplate.csproj]: @weterbogov fixed

**Review by mikitabut — APPROVED** [review]

**Pull request merged**